openreview/conference/builder.py

```diff
@@ -953,6 +954,7 @@ def create_blind_submissions(self, hide_fields=[], number=None):
 
                 revision_note = self.client.post_note(openreview.Note(
                     invitation = f'{self.support_user}/-/{self.venue_revision_name}',
+                    odate = online_date if note.odate is None else None,
```

Comment:
Shouldn't this be
```suggestion
                    odate = online_date if note.odate is not None else None,
```

Reply:
or
```suggestion
                    odate = online_date if note.odate is None else note.odate,
```

Reply:
I don't want to override the odate if already exists. So, if I pass a null in the reference, the value will be overwritten to null?  (I should write a test)

Reply:
Oh I think that's why I was checking for undefined and not undefined and null. So that you could remove the pdate/odate if you passed null to the original note

Reply:
ok, so I will accept your suggestion and pass the odate.